drop_potion_example

```diff
@@ -0,0 +1,29 @@
+Potion:
+
+"drop_item_list": [
```

Comment:
Does this file need to be in the repo?  I've generally avoided adding JSON to the repo because you can get it by just playing the game.  If it is going to be in the repo, it should probably be under a top level folder called json_samples or something similar and the entire file should just be nothing but json.  So like remove the Potion: line and just make it a complete json sample.  You could use something like this if you want to write unit tests, for example.